docs/docs/tracing/index.mdx

```diff
@@ -0,0 +1,203 @@
+---
+
+import { APILink } from "@site/src/components/APILink";
+import { Card, CardGroup, SmallLogoCard } from "@site/src/components/Card";
+import TOCInline from "@theme/TOCInline";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# MLflow Tracing for LLM Observability
+
+**MLflow Tracing** is a feature that enhances LLM observability in your Generative AI (GenAI) applications by capturing detailed information about the execution of your application's services.
+Tracing provides a way to record the inputs, outputs, and metadata associated with each intermediate step of a request, enabling you to easily pinpoint the source of bugs and unexpected behaviors.
+
+![Tracing Gateway Video](/images/llms/tracing/tracing-top.gif)
+
+
+## Why Choose MLflow?
+
+* **🪽 Free and Open** - MLflow is open source and 100% FREE. You don’t need to pay additional SaaS costs to add observability to your GenAI stack. Your trace data is hosted on your own infrastructure.
+
+
+* **🥇 Standard** - MLflow Tracing is compatible with **OpenTelemetry**, an industry-standard observability spec. You can export your trace data to various services in your existing observability stack, such as Grafana, Prometheus, Datadog, New Relic, and more.
+
+* **🤝 Framework Support** - MLflow Tracing integrates with 15+ GenAI libraries, including OpenAI, LangChain, LlamaIndex, DSPy, and others. See the [Automatic Tracing](#automatic-tracing) section for the full list of supported libraries.
+
+* **🔄 End-to-End** - MLflow is designed for managing the end-to-end machine learning lifecycle. With its model tracking and evaluation capabilities, MLflow empowers you to leverage your trace data fully.
+
+* **👥 Community** - MLflow boasts a vibrant Open Source community as a part of the Linux Foundation. With 19,000+ GitHub Stars and 15MM+ monthly downloads, MLflow is a trusted standard in the MLOps/LLMOps ecosystem.
+
+
+## Introduction to Observability and Traces
+
+:::tip
+
+If you are new to the tracing or observability concepts, we recommend starting with the [Tracing Concepts](./concept) page.
+
+:::
+
+Traces empowers your throughout the end-to-end lifecycle of a machine learning project. Click on the tabs below to see how it helps you at each step of the workflow in more details.
+
+<Tabs>
+  <TabItem value="debugging" label="Debugging" default>
+    <div class="flex-column">
+
+      <div class="flex-row">
+        <div class="flex-item">
+
+          #### Complete Debugging Experience in Your IDE or Notebook
+
+          MLflow's tracing capabilities provide deep insights into what happens beneath the abstractions of GenAI libraries, helping you precisely identify where issues occur.
+
+          You can navigate traces seamlessly **within** your preferred IDE or notebook, eliminating the hassle of switching between multiple tabs or searching through an overwhelming list of traces.
+
+
+        </div>
+
+        <div class="flex-item padding-md">
```

Comment:
note: These videos are placeholder. Planning to take those once the notebook rendering is available.